src/components/Text/index.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from './styles.css';
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  variant?: 'small' | 'normal' | 'medium' | 'large';
```

Comment:
maybe type of variant should be extracted from styles.css.ts? Sth like `variant?: keyof typeof styles`? With small modification with structure of styles object?

Reply:
Ok